pycbc/strain.py

```diff
@@ -367,10 +370,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
         else:
             logging.info("Making colored noise")
             from pycbc.noise.reproduceable import colored_noise
+            lowfreq = opt.strain_high_pass if opt.strain_high_pass else 10.
```

Comment:
@titodalcanton @micamu Maybe we should change this now to be the low frequency cutoff? Does that feel more consistent? The point is we need some frequency below (although at might be ok) the flow. Either we need a new explicit option, or we use something they should already be setting which is consistent.

Reply:
We could do something like
`lowfreq = opt.strain_high_pass if opt.strain_high_pass else opt.low_frequency_cutoff/2`
for instance?

Reply:
That doesn't seem crazy. I'm honestly not sure the best solution, but that's the best I've seen.

Reply:
Or just `lowfreq = opt.low_frequency_cutoff/2` for the colored_noise if it needs to be lower, and just not use the strain-high-pass option at all